pom.xml

```diff
@@ -97,7 +97,7 @@
         <slf4j.version>1.6.6</slf4j.version>  <!-- used for java.util.logging jul-to-slf4j interception -->
         <guava.version>17.0</guava.version>
         <xstream.version>1.4.7</xstream.version>
-        <fasterxml.jackson.version>2.7.0</fasterxml.jackson.version>  <!-- more recent jackson, but not compatible with old annotations! -->
+        <fasterxml.jackson.version>2.4.5</fasterxml.jackson.version>  <!-- more recent jackson, but not compatible with old annotations! -->
```

Comment:
What about the resteasy dependency? With 3.0.14.FINAL we get this in brooklyn-tosca:
```
java.lang.NoSuchMethodError: com.fasterxml.jackson.databind.util.BeanUtil.okNameForSetter(Lcom/fasterxml/jackson/databind/introspect/AnnotatedMethod;Z)Ljava/lang/String;
        at com.fasterxml.jackson.module.jaxb.JaxbAnnotationIntrospector.findNameForDeserialization(JaxbAnnotationIntrospector.java:1004)
        at com.fasterxml.jackson.databind.introspect.AnnotationIntrospectorPair.findNameForDeserialization(AnnotationIntrospectorPair.java:577)
```
3.0.8.FINAL was the previous version.